Release 0.8.0 fails to build with gcc 8.3.0

```sh
$cmake --build ../build/
Consolidate compiler generated dependencies of target xpload
[ 12%] Building CXX object src/CMakeFiles/xpload.dir/configurator.cpp.o
[ 25%] Building CXX object src/CMakeFiles/xpload.dir/fetch.cpp.o
[ 37%] Linking CXX static library libxpload.a
[ 37%] Built target xpload
[ 50%] Built target xpl
Consolidate compiler generated dependencies of target test_xpload
[ 62%] Building CXX object test/CMakeFiles/test_xpload.dir/test_xpload.cpp.o
[ 75%] Linking CXX executable test_xpload
../src/libxpload.a(configurator.cpp.o): In function `std::filesystem::__cxx11::path::path(std::filesystem::__cxx11::path&&)':
configurator.cpp:(.text._ZNSt10filesystem7__cxx114pathC2EOS1_[_ZNSt10filesystem7__cxx114pathC5EOS1_]+0x53): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(configurator.cpp.o): In function `std::filesystem::__cxx11::path::clear()':
configurator.cpp:(.text._ZNSt10filesystem7__cxx114path5clearEv[_ZNSt10filesystem7__cxx114path5clearEv]+0x20): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(configurator.cpp.o): In function `std::filesystem::__cxx11::path::path<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::filesystem::__cxx11::path>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::filesystem::__cxx11::path::format)':
configurator.cpp:(.text._ZNSt10filesystem7__cxx114pathC2INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES1_EERKT_NS1_6formatE[_ZNSt10filesystem7__cxx114pathC5INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEES1_EERKT_NS1_6formatE]+0x64): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(configurator.cpp.o): In function `std::filesystem::__cxx11::path::path<char [22], std::filesystem::__cxx11::path>(char const (&) [22], std::filesystem::__cxx11::path::format)':
configurator.cpp:(.text._ZNSt10filesystem7__cxx114pathC2IA22_cS1_EERKT_NS1_6formatE[_ZNSt10filesystem7__cxx114pathC5IA22_cS1_EERKT_NS1_6formatE]+0x5e): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(fetch.cpp.o): In function `std::filesystem::__cxx11::path::path(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >&&, std::filesystem::__cxx11::path::format)':
fetch.cpp:(.text._ZNSt10filesystem7__cxx114pathC2EONSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEENS1_6formatE[_ZNSt10filesystem7__cxx114pathC5EONSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEENS1_6formatE]+0x4f): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(fetch.cpp.o): In function `std::filesystem::__cxx11::path::operator/=(std::filesystem::__cxx11::path const&)':
fetch.cpp:(.text._ZNSt10filesystem7__cxx114pathdVERKS1_[_ZNSt10filesystem7__cxx114pathdVERKS1_]+0x3d): undefined reference to `std::filesystem::__cxx11::path::has_filename() const'
fetch.cpp:(.text._ZNSt10filesystem7__cxx114pathdVERKS1_[_ZNSt10filesystem7__cxx114pathdVERKS1_]+0x98): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
../src/libxpload.a(fetch.cpp.o): In function `std::filesystem::__cxx11::path::is_absolute() const':
fetch.cpp:(.text._ZNKSt10filesystem7__cxx114path11is_absoluteEv[_ZNKSt10filesystem7__cxx114path11is_absoluteEv]+0x14): undefined reference to `std::filesystem::__cxx11::path::has_root_directory() const'
../src/libxpload.a(fetch.cpp.o): In function `std::filesystem::exists(std::filesystem::__cxx11::path const&)':
fetch.cpp:(.text._ZNSt10filesystem6existsERKNS_7__cxx114pathE[_ZNSt10filesystem6existsERKNS_7__cxx114pathE]+0x14): undefined reference to `std::filesystem::status(std::filesystem::__cxx11::path const&)'
../src/libxpload.a(fetch.cpp.o): In function `std::filesystem::__cxx11::path::path<char [1], std::filesystem::__cxx11::path>(char const (&) [1], std::filesystem::__cxx11::path::format)':
fetch.cpp:(.text._ZNSt10filesystem7__cxx114pathC2IA1_cS1_EERKT_NS1_6formatE[_ZNSt10filesystem7__cxx114pathC5IA1_cS1_EERKT_NS1_6formatE]+0x5e): undefined reference to `std::filesystem::__cxx11::path::_M_split_cmpts()'
collect2: error: ld returned 1 exit status
gmake[2]: *** [test/test_xpload] Error 1
gmake[1]: *** [test/CMakeFiles/test_xpload.dir/all] Error 2
gmake: *** [all] Error 2
```

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -1,9 +1,9 @@
 add_executable(test_xpload test_xpload.cpp)
-target_link_libraries(test_xpload xpload CURL::libcurl)
+target_link_libraries(test_xpload xpload CURL::libcurl stdc++fs)
 target_include_directories(test_xpload PRIVATE ${CMAKE_SOURCE_DIR}/include)
 
 add_executable(test_xpload_rand test_xpload_rand.cpp)
-target_link_libraries(test_xpload_rand xpload CURL::libcurl)
+target_link_libraries(test_xpload_rand xpload CURL::libcurl stdc++fs)
 target_include_directories(test_xpload_rand PRIVATE ${CMAKE_SOURCE_DIR}/include)
 
 add_test(NAME "test_xpload_1" COMMAND bash -c "./test_xpload -t tag_1")

